source/ncengine/graphics2/diligent/resource/PostProcessDepthSinkBufferResource.cpp

```diff
@@ -27,9 +28,26 @@ auto PostProcessDepthSinkBufferResource::Add(Diligent::IRenderDevice& device,
         throw NcError{"Max texture count exceeded"};
     }
 
-    m_depthRenderTargets.reserve(m_depthRenderTargets.size() + numDepthRenderTargets);
-    m_depthRenderTargetViewsRT.reserve(m_depthRenderTargetViewsRT.size() + numDepthRenderTargets);
-    m_depthRenderTargetViewsSR.reserve(m_depthRenderTargetViewsSR.size() + numDepthRenderTargets);
+    if (numSamples > 1)
```

Comment:
Maybe the same idea as suggested in color. Not needed now, but at a glance, I think almost all of this code is shared with the color variety. Probably a good candidate for extracting a base class.